tests/baselines/reference/moduleResolutionWithoutExtension8.errors.txt

```diff
@@ -1,13 +1,8 @@
-error TS2468: Cannot find global value 'Promise'.
-/src/bar.cts(2,1): error TS2712: A dynamic import call in ES5/ES3 requires the 'Promise' constructor.  Make sure you have a declaration for the 'Promise' constructor or include 'ES2015' in your '--lib' option.

```

Comment:
Another example of the implicit module setting now being reflected in tests; this was defaulted to ES5, but now it's esnext.